app/move/views/_includes/incorrect-nomis-details.njk

```diff
@@ -0,0 +1,23 @@
+{% set html %}
+  <p>
+    {{ t("assessment::retrieved_from_nomis.details.content.lede") }}
+  </p>
+
+  <ul>
+    <li>
+      {{ t("assessment::retrieved_from_nomis.details.content.step_one") }}
```

Comment:
Maybe it's because I am not terribly familiar with the subject, but 1/2/3 is pretty inexpressive to me. Is there a way we can individuate more clearly what's going on in this template?

Reply:
Hmm. Good point. The content here is split into 3 steps the user must follow to update the content in another system. Those may be tweaked so didn't want to tie the key name to specifically to the content in this sense.

Is there anything you think might be easier to understand?

Reply:
This is the current content:

![image](https://user-images.githubusercontent.com/3327997/80482757-b37c2980-894c-11ea-9aaa-40b44c910fb7.png)